day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,24 @@
+for (var i=1; i<2; i++){
+  console.log( i + " speckled frogs sat on a log \n\ eating some most delicious bugs. \n\ One jumped in a pool where it was nice and cool \n\ then there were no more speckled frogs! ")
+}
+for (var i=2; i<3; i++) {
+  console.log( i + " speckled frogs sat on a log \n\ eating some most delicious bugs. \n\ One jumped in a pool where it was nice and cool \n\ then there was 1 speckled frog.")
+}
+for (var i=3; i<11; i++) {
+  console.log( i + " speckled frogs sat on a log \n\ eating some most delicious bugs. \n\ One jumped in a pool where it was nice and cool \n\ then there were " + (i-1) + " speckled frogs.");
+}
+
+//Counting down the speckled frogs
```

Comment:
Nice work shifting this to the reverse! No need to update here, but I encourage you to challenge yourself as you continue practicing to think about and code out alternate ways to solve this exercise.  What if you could only use one `for loop` here?  Can any of the copy be stored in variables, since a lot of it is repetitive?